internal/annotation/annotations.go

```diff
@@ -56,6 +56,7 @@ var annotationsByKind = map[string]map[string]struct{}{
 		"projectcontour.io/response-timeout":             {},
 		"projectcontour.io/retry-on":                     {},
 		"projectcontour.io/tls-minimum-protocol-version": {},
+		"projectcontour.io/tls-maximum-protocol-version": {},
```

Comment:
Please document this in https://projectcontour.io/docs/main/config/annotations/#contour-specific-ingress-annotations